libs/auth/src/env.d.ts

```diff
@@ -1,7 +1,7 @@
 declare namespace NodeJS {
   export interface ProcessEnv {
     NODE_ENV: 'development' | 'production' | 'test';
-    VX_ADMIN_PRIVATE_KEY_PASSWORD?: string;
     VX_CONFIG_ROOT?: string;
+    VX_MACHINE_PRIVATE_KEY_PASSWORD?: string;
```

Comment:
[Relevant line in vxsuite-complete-system](https://github.com/votingworks/vxsuite-complete-system/blob/89e898ca2dcf5d12bd64bd3b9efdb25c6104a252/config/read-vx-machine-config.sh#L53)